XmlSupportPkg/Library/XmlTreeLib/XmlTreeLib.c

```diff
@@ -877,14 +877,15 @@ BuildNodeList (
     //
     Status = RtlXmlNextToken (&State, &Next, FALSE);
     if (EFI_ERROR (Status)) {
-      DEBUG ((EFI_D_ERROR, "Failed to get the next token, Status = 0x%x\n", Status));
+      DEBUG ((EFI_D_ERROR, "Failed to get the next token, Status = %r\n", Status)); // MS_CHANGE

```

Comment:
The `MS_CHANGE` tags aren't needed for the mu_plus repo (since this entire repo represents mu value adds). For future reference, `MU_CHANGE` is preferred for repos that do have upstream components that we track changes against.